R/data.R

```diff
@@ -0,0 +1,35 @@
+#' EMU theses raw text data
+#'
+#' A data frame with text of theses from the EMU program.
+#' The data frame can be joined with the `emu_metadata` data frame via ID column.
+#'
+#' @format ## `emu_raw`
+#' A data frame with 83 rows and 2 columns:
+#' \describe{
+#'   \item{ID}{ID of the thesis}
+#'   \item{text_raw}{Raw text of the thesis.
+#'   The text has been anonymyzed and Latin special characters have been replaced with 26-letter alphabet counterparts}
+#'
+#' @source <https://repository.tudelft.nl/islandora/search/?collection=education>
+"emu_raw"
+
+
+#' EMU theses  metadata
+#'
+#' A data frame with metadata of theses from the EMU program.
+#' The data frame can be joined with the `emu_raw` data frame via ID column.
+#'
+#' @format ## `emu_metadata`
+#' A data frame with 83 rows and 8 columns:
+#' \describe{
+#'   \item{ID}{ID of the thesis}
+#'   \item{graduation_year}{Raw text of the thesis.}
+#'   \item{graduation_semester}{Raw text of the thesis.}
+#'   \item{title}{Title of the thesis.}
```

Comment:
I just realised that we will need `subtitle` and `full_title` as well. Or at least the `full_title` that was split by the `:` into `title` and `subtitle`. All those three fields are used in `find_meta_stopwords()`.

Reply:
Thanks @cforgaci! 

I've incorporated the changes. Please have a loook

Reply:
Here I see still `title`. Did you decide to drop `subtitle` and `full_title` or you replaced the values of `title` with `full_title`?